File: generate password/generate_password.cs
using System;
using System.Linq;

namespace generate_password
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.Write("Enter password length: ");
            int length = int.Parse(Console.ReadLine());
            string password = GeneratePassword(length);
            Console.WriteLine("Random Password: " + password);
        }

        static string GeneratePassword(int length)
        {
            string chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%^&*()_+";
            Random random = new Random();
            string password = new string(Enumerable.Repeat(chars, length)
                .Select(s => s[random.Next(s.Length)]).ToArray());

            return password;
        }
    }
}

File: generate password/generate_password.java
import java.util.Random;

public class generate_password {
    public static void main(String[] args) {
        System.out.print("Enter password length: ");
        int length = Integer.parseInt(System.console().readLine());
        String password = generatePassword(length);
        System.out.println("Random Password: " + password);
    }

    public static String generatePassword(int length) {
        String chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%^&*()_+";
        Random random = new Random();
        StringBuilder password = new StringBuilder();
        for (int i = 0; i < length; i++) {
            int index = random.nextInt(chars.length());
            password.append(chars.charAt(index));
        }
        return password.toString();
    }
}
